discover/portal/articles/03-collaboration/01-managing-documents-and-media/08-adapting-your-media-to-multiple-devices.markdown

```diff
@@ -0,0 +1,138 @@
+Adaptive Media
+
+Content is viewed on all sorts of devices these days, which all have different screen sizes, bandwidths, processing capabilities, etc. Until now, media content was delivered to these devices without considering these characteristics.
+
+Adaptive Media is an Application for Liferay Portal CE 7.0 and Liferay DXP, available for download from the [Liferay Marketplace](https://web.liferay.com/marketplace),  that will deliver your media content in the best format, based on the channel and device that will consume it.
+
+This provides faster loading times for devices with a slower connection, saving you bandwidth, and also gives you administrative control over the image quality provided.
+
+# How does it work?
+
+Portal Administrators define a set of image resolutions that they want for their devices. Images uploaded thereafter have a copy generated that is scaled to fit each of these resolutions.
+
+Adaptive Media is integrated with Documents and Media and Blogs. Images uploaded to Documents and Media or to a Blog Entry are analyzed and processed, generating a set of images with the defined resolutions. For instance, when a user is reading a blog entry on their mobile phone that contains an image, Adaptive Media checks the screen size of the phone and delivers the image that best fits the screen. 
```

Comment:
The WCM team is working on integrating AM with Journal; I'll ping you when it is done to update the document.